test/smoke/src/areas/positron/welcome/welcome.test.ts

```diff
@@ -0,0 +1,184 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2024 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+
+import { expect } from '@playwright/test';
+import { Application, Logger, PositronPythonFixtures, PositronRFixtures } from '../../../../../automation';
+import { installAllHandlers } from '../../../utils';
+
+/*
+ *  Welcome test cases.
+ */
+
+export function setup(logger: Logger) {
+	describe('Welcome Page', () => {
+		installAllHandlers(logger);
+
+		beforeEach(async function () {
+			const app = this.app as Application;
+
+			await app.workbench.quickaccess.runCommand('Help: Welcome');
+			app.workbench.editors.waitForActiveEditor('Welcome');
+		});
+
+		afterEach(async function () {
+			const app = this.app as Application;
+			await app.workbench.quickaccess.runCommand('View: Close All Editors');
+		});
+
+		it('Verify Welcome page header and footer', async function () {
+			const app = this.app as Application;
+
+			await expect(app.workbench.positronWelcome.logoLocator).toBeVisible();
+			const logoBox = await (await app.workbench.positronWelcome.logoLocator.elementHandle())?.boundingBox();
```

Comment:
Can be simplified as there's a `.boundingBox()` method on a locator. See (or you could use) `PositronLayouts.boundingBoxProperty()`.  https://github.com/posit-dev/positron/blob/48203703bf158da76682c06faef4a3f7998e7fa3/test/automation/src/positron/positronLayouts.ts#L103-L122